doc/usage/extensions/math.rst

```diff
@@ -15,7 +15,8 @@ Math support for HTML outputs in Sphinx
    So mathbase extension is no longer needed.
 
 Since mathematical notation isn't natively supported by HTML in any way, Sphinx
-gives a math support to HTML document with several extensions.
+gives a math support to HTML document with several extensions.  These use the
+Sphinx :rst:dir:`directives <math>` and :rst:role:`roles <math>`.
```

Comment:
`math` directive and role are not Sphinx's original. They come from reStructuredText. So "Sphinx" word is not needed here.
http://docutils.sourceforge.net/docs/ref/rst/directives.html#math
http://docutils.sourceforge.net/docs/ref/rst/roles.html

Reply:
Very true, I remotely started thinking of this a little bit too late.  Fixed.